internal/pkg/api/server_test.go

```diff
@@ -46,40 +43,38 @@ func Test_server_Run(t *testing.T) {
 	cfg.Port = port
 	addr := cfg.BindEndpoints()[0]
 
-	verCon := mustBuildConstraints("8.0.0")
-	c, err := cache.New(config.Cache{NumCounters: 100, MaxCost: 100000})
-	require.NoError(t, err)
-	bulker := ftesting.NewMockBulk()
-	pim := mock.NewMockMonitor()
-	pm := policy.NewMonitor(bulker, pim, config.ServerLimits{PolicyLimit: config.Limit{Interval: 5 * time.Millisecond, Burst: 1}})
-	bc := checkin.NewBulk(nil)
-	ct := NewCheckinT(verCon, cfg, c, bc, pm, nil, nil, nil, nil)
-	et, err := NewEnrollerT(verCon, cfg, nil, c)
-	require.NoError(t, err)
-
-	srv := NewServer(addr, cfg, ct, et, nil, nil, nil, nil, fbuild.Info{}, nil, nil, nil, nil, nil)
-	errCh := make(chan error)
+	srv := NewServer(addr, cfg, nil, nil, nil, nil, nil, nil, fbuild.Info{}, nil, nil, nil, nil, nil)
```

Comment:
@michel-laterman may be you can add a little more details on what was causing the flakyness, why do  we need create the server without bulker/policy monitor, ..?

Reply:
bulker, policy monitor, status checker are not needed in this test case because we don't test any endpoints